plank/go.mod

```diff
@@ -0,0 +1,32 @@
+module github.com/vmware/transport-go/plank
+
+go 1.16
+
+require (
+	github.com/cpuguy83/go-md2man/v2 v2.0.1 // indirect
+	github.com/fatih/color v1.12.0
+	github.com/go-git/go-git/v5 v5.4.2
+	github.com/go-stomp/stomp v2.1.4+incompatible // indirect
+	github.com/google/uuid v1.3.0
+	github.com/gorilla/handlers v1.4.2
+	github.com/gorilla/mux v1.8.0
+	github.com/gorilla/websocket v1.4.2 // indirect
+	github.com/magefile/mage v1.11.0 // indirect
+	github.com/mattn/go-isatty v0.0.13 // indirect
+	github.com/prometheus/client_golang v1.6.0
+	github.com/russross/blackfriday/v2 v2.1.0 // indirect
+	github.com/shurcooL/httpfs v0.0.0-20190707220628-8d4bc4ba7749 // indirect
+	github.com/shurcooL/vfsgen v0.0.0-20200824052919-0d455de96546
+	github.com/sirupsen/logrus v1.8.0
+	github.com/streadway/amqp v1.0.0
+	github.com/stretchr/objx v0.2.0 // indirect
+	github.com/stretchr/testify v1.7.0
+	github.com/urfave/cli v1.22.5
+	github.com/vmware/transport-go v1.2.0
+	golang.org/x/net v0.0.0-20210805182204-aaa1db679c0d
+	golang.org/x/sys v0.0.0-20210630005230-0f9fa26af87c // indirect
+	golang.org/x/tools v0.1.0 // indirect
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
+)
+
+replace github.com/vmware/transport-go => ../
```

Comment:
Is this supposed to be here?

Reply:
i was a little on the fence on this one, so i'm open to discussion because it's up in the air depending on how we want the dev experience to be. by having plank reference its local parent folder for the `transport-go` dependency instead of a tag version, making changes to its parent project (just like this PR) becomes a lot streamlined; the developers wouldn't need to make any changes to go.mod while working on their Plank PR that involves changes to its parent project.

on the other hands, by NOT having the replace directive, we are tacitly forcing PRs to be split between Plank and Transport. in the context of this particular PR though, it would be somewhat a mild inconvenience for me as the changes to Transport code are mostly to fill in gaps that were found while working on this PR :)

let me know what you think

Reply:
leave it as is, makes sense to me.